extensions/amp-story/1.0/amp-story-share-menu.js

```diff
@@ -221,13 +229,12 @@ export class ShareMenu {
   /**
    * Handles click events and maybe closes the menu for the fallback UI.
    * @param  {!Event} event
-   * @return {*} TODO(#23582): Specify return type
    */
   onShareMenuClick_(event) {
     const el = dev().assertElement(event.target);
 
     if (el.classList.contains('i-amphtml-story-share-menu-close-button')) {
-      return this.close_();
+      this.close_();
```

Comment:
Looks like there was a PR that got merged in the meantime.  Can you update to add back the return and its jsdoc?

Reply:
Hmmm I actually deleted those lines as they weren't needed. Should I remove them in a sep PR?

Reply:
Whoops my bad, you're absolutely right!  Thought this was related to #23559, but (a) it's not, and (b) that's been merged for a while now.